src/writeData/utils/updateTelegrafPlugins.mjs

```diff
@@ -321,44 +326,6 @@ const fetch = (path, parsedPath) => {
   })
 }
 
-const formatConfigurationText = configurationText => {
-  const configurationLines = configurationText.split('\n')
-  const isCommented = configurationLines.every(
-    line => line[0] === '#' || line === ''
-  )
-
-  if (isCommented) {
-    return (
-      configurationLines
-        .map(text => {
-          let sliceCounter = 0
-          if (text[0] === '#') {
-            sliceCounter += 1
-          }
-          if (text[1] === ' ') {
-            sliceCounter += 1
-          }
-          return text.slice(sliceCounter)
-        })
-        .join('\n') + '\n'
-    )
-  }
-  return configurationText + '\n'
-}
-
-const formatReadmeText = readmeText => {
-  // Change all relative links to Github links
-  const relativeLink = /\(\/plugins\//gi
-  const replacement =
-    '(https://github.com/influxdata/telegraf/tree/master/plugins/'
-
-  return readmeText
-    .map(line => {
-      return line.replace(relativeLink, replacement)
-    })
-    .join()
-}
-
```

Comment:
Moved into another file because of the `.mjs` extension here, which makes it annoyingly difficult to write Jest unit tests.

Reply:
Ended up moving this back into this file but at a different location.